src/Plane.js

```diff
@@ -10,13 +10,25 @@ class Plane {
     noTakeOffRequired;
     noLandingRequired;
 
-    constructor(flightNumber, takenOff = true) {
+    constructor(flightNumber, takenOff) {
         this.flightNumber = flightNumber;
         this.takenOff = takenOff; 
     }
 
+    takeOffPlane() {
+        
+        const planeIndex = airport.airportPlanes.indexOf(plane.flightNumber);
+
+        if (airport.airportPlanes.includes(this.flightNumber)) {
+            (airport.airportPlanes.splice(planeIndex, 1))
+        }
+            if (airport.airportPlanes.splice(planeIndex, 1)) {
```

Comment:
There's definitely a better way to write this rather than repeating that line of code!